helix-core/tests/path.rs

```diff
@@ -0,0 +1,114 @@
+use std::{
+    env::set_current_dir,
+    error::Error,
+    path::{Component, Path, PathBuf},
+};
+
+#[cfg(unix)]
+use std::os::unix;
+#[cfg(windows)]
+use std::os::windows;
+
+use helix_core::path::get_normalized_path;
+use tempfile::Builder;
+
+#[test]
+fn test_normalize_path() -> Result<(), Box<dyn Error>> {
```

Comment:
* Added in d9da1026bdd3ca172b78ee696dbefca1af0e06d4
* Modified my own test c20d041ab612773c8f44d701f9d717c31c3bb08b for `alt-ret` a bit, because in this test I was using a global search picker that uses `WalkDir` with `following_symlinks` under the hood, now I compare by filename instead of full path.

I'm also wondering is it bad at adding a crate with a different version than the version in the other workspace packages. I added the temporary file "3.7.0" to `helix-core`, but `helix-term` uses "3.7.1". Will dependabot equates them?

Reply:
I had to drop the the for alt-ret in #7814 anyway because we will stream in picker results there so that test is always flakey (its essentially a race condition whether testcode or the streaming is faster). I can't really thing of a way to solve that so we decided to drop the test. I don't think you need to fix it here.

The version missmatche doesn't matter as as long as they are semver compatible Cargo will unify them anyway (if you check the `Cargo.lock` changes there is no actual new dependency)

Reply:
If I revert the test fix, the action will fail. If your PR merges first, I'll need just resolve one merge conflict.